### ISSUE TITLE
Clicking the videos does nothing

### ISSUE THREAD

**Issue by Kekun** [Tubefeeder/Tubefeeder#112]
I added a few PeerTube and YouTube channels, and clicking the videos from both platforms does nothing. I'm not sure what it's supposed to do in the first place.

**Comment by Schmiddiii** [comment]
You are supposed to click the videos and they should play, so you did not misunderstand anything.
Have you set up a video player (by default `mpv`)?

Some further documentation:

- [General wiki](http://www.tubefeeder.de/wiki/)
- [Setting up a video player](https://www.tubefeeder.de/wiki/different-player.html)

**Comment by Kekun** [comment]
Ah, I had no idea the app was playing the video in another app I had to set that way. Same for the downloader.

Is it planned to replace that by playing the video in the app, by embedding the downloader in the flatpak and by dropping these options?

**Comment by Schmiddiii** [comment]
No,  I will probably not do so. There are so many good video players and different opinions about video players out there that any video player I will integrate will most likely be worse than the choise that anyone can do themselves.

I would assume your problem is now solved, so I will close this now.

**Comment by Kekun** [comment]
Kinda, I know know that Tubefeeder isn't the app I'm looking for, thanks! :)

**Comment by Newbytee** [comment]
I think this should be reopened. While I understand not wanting to build your own video player, I think the UX could be better. How about checking if opening a video player failed, and if so, show a pop-up telling the user to install a video player?

**Comment by Newbytee** [comment]
Never mind, the aforementioned error was because manually I had denied Tubefeeder the necessary permission to use the `org.freedesktop.Flatpak portal`. Now it tells me that it can't find mpv, but it's still only printed to the terminal, not shown in the GUI.

**Comment by Schmiddiii** [comment]
I agree that this could be shown in the UI.
